project-reports/2023/2023-Q3-Hyperledger-FireFly.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: default
+title: 2023 Q3 Hyperledger FireFly
+parent: 2023
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+---
+
+# Project Health
+
+Hyperledger FireFly is applying to move to Graduated Project status. A separate document will be submitted to the TOC to propose this change. Throughout the past year, there has been very strong activity within the project, and Hyperledger FireFly has continued to gain even more momentum this quarter. FireFly is powering many use cases in the blockchain and digital assets community, with recent examples that include Swift’s phase 2 CBDC work with 32 central and global transaction banks across 4 use cases, and the DFCRC and Reserve Bank of Australia CBDC work running 15 projects including ones with ANZ Bank and Mastercard.
+
+Hyperledger published the case study [The Hyperledger FireFly Story: Kaleido Taps into the Hyperledger Community to Develop Next-Gen Solutions](https://media.licdn.com/dms/document/media/D561FAQFHEjkdNUvJEg/feedshare-document-pdf-analyzed/0/1692713935821?e=1694044800&v=beta&t=KLpydg2_L7VbFMuSJpm3OsnKUJbxw3Q6dDzto_UPFGE).
+
+We have onboarded 3 new maintainers to the project and one maintainer has transitioned to a new company. These new maintainers include Matthew Whitehead & Chengxuan Xing from Kaleido and Dzianis Andreyenka from Instincttools. Vinod Damle has moved to Fidelity Investments and will continue as a maintainer.
```

Comment:
A couple of questions that will probably be asked during the request to graduate:
1. For the maintainer that transitioned to a new company, does this person's day-to-day involve work with FireFly?
2. For the maintainer from Instincttools, what is the level of maintainership?

Reply:
Thanks. These are good questions and I would be happy to chat about them in detail.